src/style.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  --border-color: #e1e7ed;
+  --background-color: #fff;
+  --dark-color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 700;
+}
+
+.html {
+  font-family: "Avenir", Arial, Calibri, sans-serif;
+  font-size: 16px;
+}
+
+.search-bar__form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-bar__form::before {
+  content: "";
+  position: absolute;
+  background-image: url(images/Search.svg);
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.search-bar__form--big::before {
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar__form--small::before {
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
+
+.search-bar__input {
+  width: 100%;
+  box-sizing: border-box;
+  font-family: "Avenir", Arial, sans-serif;
```

Comment:
I don't think that you need to specify `font-family` here. you already did it for all html above